setup.py

```diff
@@ -60,7 +60,7 @@
         'zope.processlifetime',
         'zope.schema',
         'zope.testing',
-        'Zope >= 4.0b1',
+        'Zope > 4.0b2',
```

Comment:
This should probably be a `>= 4.0b3`, for clarity.

Reply:
Currently the version on Zope master is 4.0b3.dev0. IIRC that `>= 4.0b3` wouldn't match that, as the dev0 version of something is always less than the release it refers to (even if that is a alpha/beta). So by specifying `>= 4.0b3` the tests for ZServer currently wouldn't install, as they couldn't find a compatible Zope version.

Or do I misremember this?

Reply:
@hannosch You remember right I did not want to write `>=  4.0b3.dev0`